WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Service/BlogDashboardService.swift

```diff
@@ -14,19 +14,19 @@ class BlogDashboardService {
     /// Fetch cards from remote
     func fetch(blog: Blog, completion: @escaping (DashboardSnapshot) -> Void, failure: (() -> Void)? = nil) {
 
-        guard let dotComID = blog.dotComID?.intValue else {
+        guard let wpComID = blog.dotComID?.intValue else {
             return
         }
 
         let cardsToFetch: [String] = DashboardCard.remoteCases.map { $0.rawValue }
 
-        remoteService.fetch(cards: cardsToFetch, forBlogID: dotComID, success: { [weak self] cardsDictionary in
+        remoteService.fetch(cards: cardsToFetch, forBlogID: wpComID, success: { [weak self] cardsDictionary in
```

Comment:
Maybe we should use `dotComID` everywhere just to be consistent with the `Blog` entity?

Reply:
Fixed! 970aa7f